### PR TITLE
CFP for research plan and 2019 report

### PR DESCRIPTION
**Pull request by ultrasaurus**
@iisa wrote first draft, review and edits with @mxie and Jéneen in meeting today

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@mxie pls review and comment on typos or suggested improvements.  I marked it as a draft, so feel free to merge if you prefer to review live (or just commenting is fine too).

File: _posts/2020-01-07-call-for-proposal-research-plan.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+just about the physical space for gatherings or funding more diverse
+participation at conferences. We also foster communities where people can feel
+like they belong in an industry with systemic bias against most people.
+
+Over the past decade, we have grown as a volunteer-driven grassroots movement.
+This is a challenging feat and we are very proud of how each of us has played a
+small part in creating positive impact across the industry.  We want to continue
+facilitating workshops, writing, speaking, and making change until technology is
+equally available and accessible to everyone. We also want to dig deeper into
+why our model is successful and the impact it has on our communities, industry,
```
Comment (mxie) [inline _posts/2020-01-07-call-for-proposal-research-plan.md]:
```suggestion
why our model is successful and how it impacts our communities, industry,
```

File: _posts/2020-01-07-call-for-proposal-research-plan.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+with growth of number of events over time and across geographies.  We have
+anecdotal evidence that there is significant impact with repeat participation.
+We also have other activities, like podcasts, online events and even a book in
+the works!  In order to sustain our model, we have organically created
+leadership pipelines where students become teachers and take on other leadership
+roles.  Repeat participation also causes volunteers to bring positive change to
+their workplaces.  By gaining more insight into the space that we are creating
+through these workshops, we believe that we can hone our practices, improving
+the experience of working within our communities and increasing our positive
+impact on the world.We would like to hire an individual researcher or firm to
```
Comment (mxie) [inline _posts/2020-01-07-call-for-proposal-research-plan.md]:
```suggestion
impact on the world.   We would like to hire an individual researcher or firm to
```

Comment (mxie) [reply _posts/2020-01-07-call-for-proposal-research-plan.md]: Maybe even in a new paragraph?

File: _posts/2020-01-07-call-for-proposal-research-plan.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+a 2020 research plan.
+
+Project Scope:
+
+1. Research and write an impact report for the 2019 fiscal year. Our expectation
+is that report would include no more than 2-3 pages of narrative text, with an
+appendix of data gathering activities, which would include:
+  * Reviewing current and past data (participation in workshops and other programs)
+  * Interviewing board members and staff (5 people)
+  * Performing a qualitative assessment, such as a survey with follow-up interviews with participants *   (volunteer leaders, students, teachers, etc)
```
Comment (mxie) [inline _posts/2020-01-07-call-for-proposal-research-plan.md]:
```suggestion
  * Performing a qualitative assessment, such as a survey with follow-up interviews with participants (volunteer leaders, students, teachers, etc)
```

**Pull request merged**